testcases/open_posix_testsuite/conformance/interfaces/fscanf/1-1.c

```diff
@@ -0,0 +1,109 @@
+/*
+ * This file is licensed under the GPL license. For the full content
+ * of this license, see the COPYING file at the top level of this
+ * source tree.
+ */
+
+/*
+ * assertion:
+ *  When there are excess arguments as compared to the directives in the
+ *	control string format, they will be ignored.
+ *
+ * method:
+ *  -open file in write mode
+ *  -write 3 integers into the file
+ *  -close the file
+ *  -open the same file in read mode
+ *  -read only 2 integers from the file using fscanf() while giving 3 pointer
+ *   arguments,
+ *  -third integer value will be ignored
+ *  -check the 2 read integer values
+ */
+
+#define _XOPEN_SOURCE 700
```

Comment:
FYI:
https://patchwork.ozlabs.org/patch/1008664/#2047324
http://lists.linux.it/pipermail/ltp/2018-December/010211.html
http://lists.linux.it/pipermail/ltp/2018-December/010231.html